src/templates/app/user/statement.jsx

```diff
@@ -26,14 +27,29 @@ const Statement = () => (
                 <AccountStatisticsBox id='total_withdrawals' heading={it.L('Total withdrawals')} />
                 <AccountStatisticsBox id='net_deposits'      heading={it.L('Net deposits')} />
             </div>
-
             <div id='util_row' className='gr-row gr-padding-10 gr-parent invisible container'>
-                <div className='gr-12 gr-12-m'>
-                    <div className='gr-row gr-row-align-right gr-row-align-left-m gr-row-align-middle'>
+                <div className='gr-8'>
+                    <div className='gr-row gr-row-align-left gr-row-align-left-m'>
```

Comment:
Since you changed alignment to left, you can remove the mobile class as well. They both set the same alignment.